### DISCUSSION TITLE
How to merge two graph objects?

### DISCUSSION THREAD

**Discussion by TonyinBio** [graphology/graphology#462] [Q&A] [answered]
Hi,

I was wondering how to combine two graphology graphs of the same type into one

Thank you

**Comment by Yomguithereal** [comment, marked as answer]
There are many ways to combine two graphs. Check for instance the binary operators [here](https://graphology.github.io/standard-library/operators), such as `union` and `disjointUnion`.

**Reply by TonyinBio** [reply]
Wow, thank you so much! I did not see that in the documentation, lol
